### PR TITLE
feat(service-portal): Inbox UI updates

### PR DESCRIPTION
**Pull request by thordurhhh**
## What

Update inbox UI.

* Add filters for unread
* Add sorting

## Why

Added filter/sorting value for users.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/service-portal/documents/src/screens/Overview/Overview.tsx [hunk 1]
```diff
@@ -178,6 +313,9 @@ export const ServicePortalDocuments: ServicePortalModuleComponent = ({
   const { data: orgData } = useQuery(GET_ORGANIZATIONS_QUERY)
   const organizations = orgData?.getOrganizations?.items || {}
 
+  console.log('categories', categories)
```
Comment (disaerna) [inline libs/service-portal/documents/src/screens/Overview/Overview.tsx:316]: 👀 👀 👀

File: libs/service-portal/documents/src/screens/Overview/Overview.tsx [hunk 2]
```diff
@@ -199,68 +337,120 @@ export const ServicePortalDocuments: ServicePortalModuleComponent = ({
...
-                    defaultValue={categories[0]}
-                    options={categories}
-                    value={filterValue.activeCategory}
-                    onChange={handleCategoryChange}
-                    label={formatMessage({
+          <Filter
+            resultCount={0}
+            variant={'popover'}
+            align="right"
+            labelClear={'Hreinsa síu'}
```
Comment (disaerna) [inline libs/service-portal/documents/src/screens/Overview/Overview.tsx:344]: Tranlsations missing

Comment (disaerna) [reply libs/service-portal/documents/src/screens/Overview/Overview.tsx:344]: ... on all labels for <Filter

File: libs/service-portal/documents/src/screens/Overview/Overview.tsx [hunk 3]
```diff
@@ -199,68 +337,120 @@ export const ServicePortalDocuments: ServicePortalModuleComponent = ({
...
+              display="flex"
+              flexDirection="column"
+              justifyContent="spaceBetween"
+              paddingX={3}
+              className={styles.unreadFilter}
+            >
+              <Box paddingY={3}>
+                <Checkbox
+                  id="show-unread"
+                  label="Sýna einungis ólesið"
```
Comment (disaerna) [inline libs/service-portal/documents/src/screens/Overview/Overview.tsx:371]: Translations

File: libs/service-portal/documents/src/screens/Overview/Overview.tsx [hunk 4]
```diff
@@ -199,68 +337,120 @@ export const ServicePortalDocuments: ServicePortalModuleComponent = ({
...
+                />
+              </Box>
+              <Box
+                borderBottomWidth="standard"
+                borderColor="blue200"
+                width="full"
+              />
+            </Box>
+            <FilterMultiChoice
+              labelClear="Hreinsa val"
```
Comment (disaerna) [inline libs/service-portal/documents/src/screens/Overview/Overview.tsx:383]: Translations

File: libs/service-portal/documents/src/screens/Overview/Overview.tsx [hunk 5]
```diff
@@ -199,68 +337,120 @@ export const ServicePortalDocuments: ServicePortalModuleComponent = ({
...
-                    onClick={handleDateRangeButtonClick}
+                    }),
+                    selected: [...filterValue.activeCategories],
+                    filters: categories,
+                    inline: false,
+                    singleOption: false,
+                  },
+                  {
+                    id: 'group',
+                    label: 'Flokkur',
```
Comment (disaerna) [inline libs/service-portal/documents/src/screens/Overview/Overview.tsx:422]: Translations

File: libs/service-portal/documents/src/screens/Overview/Overview.tsx [hunk 6]
```diff
@@ -199,68 +337,120 @@ export const ServicePortalDocuments: ServicePortalModuleComponent = ({
...
+              <Box marginTop={1}>
+                <Accordion
+                  dividerOnBottom={false}
+                  dividerOnTop={false}
+                  singleExpand={false}
+                >
+                  <AccordionItem
+                    key={'category.id'}
+                    id={'category.id'}
+                    label="Dagsetningar"
```
Comment (disaerna) [inline libs/service-portal/documents/src/screens/Overview/Overview.tsx:448]: Translations

**Review by disaerna — CHANGES_REQUESTED** [review]
Minor comments, otherwise very very nice 💯 🥇

File: libs/service-portal/documents/src/screens/Overview/Overview.tsx [hunk 7]
```diff
@@ -74,20 +124,34 @@ const getFilteredDocuments = (
...
-    return filteredDocuments.filter((x) =>
+    filteredDocuments = filteredDocuments.filter((x) =>
       x.subject.toLowerCase().includes(searchQuery.toLowerCase()),
     )
   }
   return filteredDocuments
 }
 
+const getSortDirection = (direction: SortDirectionType) => {
+  return direction === 'asc' ? 'desc' : 'asc'
```
[the remark this answers is not in the document]
Comment (thordurhhh) [reply libs/service-portal/documents/src/screens/Overview/Overview.tsx:152]: Yes it is. It's actually taking the current direction in and reversing.
I will make it a bit clearer to read though. 😅

**Review by disaerna — APPROVED** [review]
👍  I like

**Pull request merged**